tests/Integration/UseCases/AddDonation/AddDonationUseCaseTest.php

```diff
@@ -168,7 +168,7 @@ private function newInvalidDonationRequest(): AddDonationRequest {
 	public function testGivenInvalidRequest_noConfirmationEmailIsSend() {
 		$mailer = $this->newMailer();
 
-		$mailer->expects( $this->never() )->method( 'sendMail' );
+		$mailer->expects( $this->never() )->method( 'sendMailFromDonation' );
```

Comment:
`$this->any()` binds to less stuff

`sendMailFromDonation` comes over as an odd name to me. You can say "from donation" when constructing a mail... in this case it kinda says the mail is coming from the donation tho. How about "for donation"? Or perhaps just `sendMail`, or `sendConfirmationMail`?